lib/ansible/modules/database/mysql/mysql_db.py

```diff
@@ -117,8 +124,11 @@
 
 
 def db_exists(cursor, db):
-    res = cursor.execute("SHOW DATABASES LIKE %s", (db.replace("_", r"\_"),))
-    return bool(res)
+    res = 0
+    db = db.replace(" ", "")
+    for each_db in db.split(','):
+        res += cursor.execute("SHOW DATABASES LIKE %s", (each_db.replace("_", r"\_"),))
```

Comment:
No sure there: why should databases with `_` in their names should be escaped ?
I'd understand for `-` but not for `_`

Reply:
Not sure since this code has been added in earlier patches and not in this patch.